webrender/src/picture.rs

```diff
@@ -232,7 +234,7 @@ impl PicturePrimitive {
         let local_content_rect = prim_run_rect.local_rect_in_actual_parent_space;
 
         match self.kind {
-            PictureKind::Image { composite_mode, ref mut real_local_rect, .. } => {
+            PictureKind::Image { composite_mode, ref mut real_local_rect, ref mut content_rect, .. } => {
```

Comment:
would it (possibly, in the future) make sense outside of `FilterOp::DropShadow`? If not, we may move it inside the `DropShadow variant` for clarity.